OWNERS_ALIASES

```diff
@@ -64,4 +64,19 @@ aliases:
     - mrhohn
     - dcbw
     - danwinship
-    - caseydavenport
\ No newline at end of file
+    - caseydavenport
+  sig-apps-reviewers:
+    - bprashanth
+    - enisoc
+    - erictune
+    - foxish
+    - janetkuo
+    - kargakis
+    - kow3ns
+    - lukaszo
+    - mfojtik
+    - smarterclayton
+    - soltysh
+  sig-apps-api-reviewers:
```

Comment:
Does not match what you add in pkg/apis/policy/OWNERS (sig-apps-api-reviewers vs sig-apps-api-approvers)